packages/wdio-sauce-service/src/launcher.js

```diff
@@ -75,11 +77,39 @@ export default class SauceLauncher {
         obs.observe({ entryTypes: ['measure'], buffered: false })
 
         performance.mark('sauceConnectStart')
-        this.sauceConnectProcess = await this.api.startSauceConnect(this.sauceConnectOpts)
+
+        try {
+            this.sauceConnectProcess = await this.startTunnel()
+        } catch (err) {
+            ++this.scStartTrials
+            /**
+             * fail starting Sauce Connect eventually
+             */
+            if (
+                /**
+                 * only fail for ENOENT errors due to racing condition
+                 * see: https://github.com/saucelabs/node-saucelabs/issues/86
+                 */
+                !err.message.includes('ENOENT') ||
+                /**
+                 * or if we reached the maximum rety count
+                 */
+                this.scStartTrials >= MAX_SC_START_TRIALS
+            ) {
+                throw err
+            }
+            log.debug(`Failed to start Sauce Connect Proxy due to ${err.stack}`)
+            log.debug(`Retrying ${this.scStartTrials}/${MAX_SC_START_TRIALS}`)
+            this.sauceConnectProcess = await this.startTunnel()
+        }
```

Comment:
I suggest to move this whole try/catch block into `startTunnel` and make sure we return at the end and because right now this will be only retried once.